### PR TITLE
ada4250: IIO support and minor updates

### PR DESCRIPTION
**Pull request by amiclaus**
- add IIO support
- minor driver updates

Signed-off-by: Antoniu Miclaus <antoniu.miclaus@analog.com>

### CODE REVIEW

File: drivers/amplifiers/ada4250/iio_ada4250.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#ifndef IIO_ADA4250_H
+#define IIO_ADA4250_H
+
+#include "iio_types.h"
+#include "ada4250.h"
```
Comment (mchindri) [inline drivers/amplifiers/ada4250/iio_ada4250.h:44]: nitpick: ada4250.h is not needed here. Just in the .c file.

**Review by mchindri — APPROVED** [review]

File: drivers/amplifiers/ada4250/iio_ada4250.h (hunk 1 above)
Comment (amiclaus) [reply drivers/amplifiers/ada4250/iio_ada4250.h:44]: Sure, I can move it into the c file.

**Comment by amiclaus** [comment]
V2:
- move include into the *.c file

**Review by amiclaus — COMMENTED** [review]
V3:
- remove extra empty line

**Pull request merged**